src/components/SamplesTableProcessingInformationPipeline.js

```diff
@@ -0,0 +1,229 @@
+import { Box, Heading, Paragraph, Text } from 'grommet'
+import isLastIndex from 'helpers/isLastIndex'
+import { links } from 'config'
+import { Anchor } from 'components/Anchor'
+import { SamplesTableProcessingInformationPipelineSteps } from 'components/SamplesTableProcessingInformationPipelineSteps'
+import { SamplesTableProcessingInformationSubmitterSupplied } from 'components/SamplesTableProcessingInformationSubmitterSupplied'
+import { SamplesTableProcessingInformationVersion } from 'components/SamplesTableProcessingInformationVersion'
+
+const Description = ({ name, description }) => (
```

Comment:
Can we update this to take a child instead of a prop for the description content.

Reply:
We'll address this in a separate issue #493.